### ISSUE TITLE
MKL does not seem to get used on Windows

### ISSUE THREAD

**Issue by twiecki** [aesara-devs/aesara#641]
From: https://github.com/pymc-devs/pymc/issues/5101#issuecomment-955009453

A user has to add 
```
[blas]
ldflags = -LC:\ProgramData\Anaconda3\Library\bin -lmkl_intel_thread -lmkl_core -lpthread -lm
```

To his config for a NumPy BLAS warning to go away.

**Comment by brandonwillard** [comment]
Can this still be reproduced?

**Comment by twiecki** [comment]
@flimao ping

@brandonwillard did we do something that might have fixed this?

**Comment by brandonwillard** [comment]
> @brandonwillard did we do something that might have fixed this?

Can't say; Conda packages&mdash;and Conda itself&mdash;changes a lot, and that's why we need to MWE (or a nearly complete environment description) in order to tell.

Without one, we can't begin to say how Aesara should be changed in order to compensate.

**Comment by flimao** [comment]
@twiecki @brandonwillard sadly I can still reproduce it.

Following the steps
1. Update conda: ```conda update -n base -c defaults conda```;
2. Follow the instructions for installing PyMC3 [here](https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(Windows)); and
3. On the resulting Python prompt, import PyMC3: ```import pymc3```,

the NumPy BLAS warning appears.

After exiting the prompt, creating the config file like @twiecki mentioned, and repeating step 3, the warning goes away.

**Comment by brandonwillard** [comment]
PyMC3 doesn't use Aesara.

**Comment by flimao** [comment]
@brandonwillard same result. 

NumPy BLAS warning after creating new conda env and installing aesara and nothing else, warning goes away after creating conf file `.aesararc` like OP and restarting the Python prompt.

Also I'm not familiar with Aesara. I posted a solution to this warning in the discussion about PyMC3 installation instructions. and then @twiecki posted it here.

**Comment by brandonwillard** [comment]
> Also I'm not familiar with Aesara. I posted a solution to this warning in the discussion about PyMC3 installation instructions. and then @twiecki posted it here.

No problem; we appreciate you trying this out for us.

**Comment by brandonwillard** [comment]
I just realized that we no longer specify `mkl` in the `aesara-feedstock` specification, so it may be necessary to `conda install mkl mkl-service aesara`.

**Comment by twiecki** [comment]
Correct, we don't force MKL anymore which doesn't work for e.g. M1 chips.

**Comment by flimao** [comment]
Same result, unfortunately.

I ran `conda create -n aesara -c conda-forge python mkl mkl-service aesara`. The NumPy BLAS warning didn't appear only after I made the config file.
